qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/HomePage.java

```diff
@@ -80,8 +80,14 @@ public Boolean isSystemAdministrationAppPresent() {
 	}
 
 	public Boolean isConfigureMetadataAppPresent() {
-		return isAppButtonPresent(CONFIGURE_METADATA_APP_ID);
+		try {
+			driver.findElement( CONFIGURE_METADATA);
```

Comment:
Why this change? Wasn't line 83 above enough?

Reply:
Resolved it .

Reply:
Resolved Thanks